include/swift/Runtime/Exclusivity.h

```diff
@@ -44,13 +44,19 @@ void swift_beginAccess(void *pointer, ValueBuffer *buffer,
 /// replacement function if it should be called.
 /// Returns null if the original function (which is passed in \p CurrFn) should
 /// be called.
+#ifdef __APPLE__
+__attribute__((weak))
+#endif
```

Comment:
This will export these symbols as a weak definition, which is probably not what you want. Did you mean `__attribute__((weak_import))` ?

Reply:
Yes. Will fix.